pkg/apis/batch/types.go

```diff
@@ -287,6 +328,17 @@ type JobSpec struct {
 	// +optional
 	PodFailurePolicy *PodFailurePolicy
 
+	// SuccessPolicy specifies the policy when the Job can be declared as succeeded.
+	// If empty, the default behavior applies - the Job is declared as succeeded
+	// only when the number of succeeded pods equals to the completions.
+	// Specified it must be immutable and works only for the Indexed Jobs.
```

Comment:
```suggestion
	// When the field is specified, it must be immutable and works only for the Indexed Jobs.
```
?
Not sure what you mean for this sentence to be honest.

Reply:
SGTM

Reply:
Done.